app/assets/stylesheets/views/article.scss

```
@@ -150,6 +150,13 @@
     flex-direction: column;
   }
 
+  // We don't show tooltips on touch-only devices
+  @media (max-width: $breakpoint-m) and (hover: hover) {
+    .crayons-tooltip__content {
+      display: none;
+    }
+  }
```

```suggestion
  @media (hover: none) {
    .crayons-tooltip__content {
      display: none;
    }
  }
```

`hover: hover` is selecting devices that _do_ have the ability to hover, so we just want to flip this. I think we can also leave out the breakpoint, so that we don't trigger this sticky content on e.g. an ipad that is bigger than breakpoint-m